src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -183,4 +188,47 @@ describe( 'ContentTabAssistant', () => {
 
 		expect( screen.queryByText( 'New message' ) ).toBeNull();
 	} );
+
+	test( 'does not render the Welcome messages and example prompts when not authenticated', () => {
+		( useAuth as jest.Mock ).mockImplementation( () => ( {
+			client: {
+				req: {
+					post: clientReqPost,
+				},
+			},
+			isAuthenticated: false,
+			authenticate,
+		} ) );
+		render( <ContentTabAssistant selectedSite={ runningSite } /> );
+		expect( screen.getByText( 'Hold up!' ) ).toBeInTheDocument();
```

Comment:
Nitpick: I see that most of the expectations use `toBeInTheDocument` to check UI elements. However, this only ensures that it's rendered not that is visible. I wonder if we should use [`toBeVisible`](https://github.com/testing-library/jest-dom?tab=readme-ov-file#tobevisible) instead.

Reply:
Good suggestion! Updated in https://github.com/Automattic/studio/pull/293/commits/595ffa2c7b0e7c26b9b034247a210497db917bd3

Reply:
Thanks for applying the suggestion 🙇 ! I think I should have been clearer in my above comment, my intent was to recommend using `toBeVisible` on every assert that an element is present. We don't need to update all in this file, so we could do it on a separate PR to avoid delaying the merge. I'll open a PR to update this.

Reply:
FYI I created the PR: https://github.com/Automattic/studio/pull/307